docs/get-started/README.md

```diff
@@ -2,7 +2,13 @@
 
 ## What is IRISnet
 
-IRIS network (a.k.a. IRISnet) is named after Greek goddess **Iris**, said to be the personification of the rainbow and the faithful messenger between heaven and humanity. IRIS network aims to build the foundation which facilitates construction of distributed business applications. IRIS hub will provide iServices, which allow resources such as data service and computing services being invoked across blockchains. To know more about IRISnet, please read this blog.
+IRIS network (a.k.a. IRISnet) is named after Greek goddess **Iris**, said to be the personification of the rainbow and the faithful messenger between heaven and humanity. IRIS network aims to build the foundation which facilitates construction of distributed business applications. IRIS hub will provide iService, which allow resources such as data service and computing services being invoked across blockchains. To know more about IRISnet, please read this blog.
+
```

Comment:
IRIS Network (a.k.a IRISnet) is named after...

IRISnet aims to provide a foundation **that** facilitates ...

IRIS Hub (a.k.a IRIShub), which is the centerpiece of IRISnet, introduces a unified service model that allows resources such as data **services** and computing services **to be** invoked ...


URL to the blog is missing.

Reply:
fixed